TedTalks/Extensions/UIImageView+URL.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIImage+URL.swift
+//  TedTalks
+//
+//  Created by Miguel Askar on 5/08/21.
+//
+
+import UIKit
+
+extension UIImageView {
+    func downloaded(from url: URL, contentMode mode: ContentMode = .scaleAspectFit) {
```

Comment:
We might want to set the content mode just once in the class that holds the `UIImageView`, for this case in `TedTalkTableViewCell`'s `awakeFormNib()`

Reply:
Ok, fixed